source/help/faq/whytex.md

```diff
@@ -1,23 +1,23 @@
 # Why Submit the TeX/LaTeX Source?
 
 1. TeX has many advantages that make it ideal as a format for the archives: It is plain text, it is compact, it is freely available for all platforms, it produces extremely high-quality output, and it retains [contextual information](#contextual).
-2. It is thus more likely to be a good source from which to generate newer formats, e.g., HTML, MathML, various ePub formats, etc.. Possession of the source thus provides many additional options for future document formats and other uses.
+2. It is thus more likely to be a good source to generate newer formats, e.g., HTML, MathML, various ePub formats, etc.. Possession of the source thus provides many additional options for future document formats and other uses.
```

Comment:
Sorry, nothing deep, I just thought it was a bit redundant since "source" already evokes an image of "from which". I have reverted this and all other preferences to now only fix typos. Thanks for your work on this.